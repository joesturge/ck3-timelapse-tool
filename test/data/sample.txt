@correct_culture_primary_score = yes
@better_than_the_alternatives_score = "no no"
@always_primary_score = 1000

# EMPIRES

# special titles
k_papal_state = {
	color = { 255 249 198 }
	color2 = { 220 220 0 }
	definite_form = yes
	ruler_uses_title_name = @better_than_the_alternatives_score
	landless = yes
	
	capital = c_roma
	
	ai_primary_priority = {
		add = @always_primary_score
	}

	destroy_if_invalid_heir = "yes"
	no_automatic_claims = "@correct_culture_primary_score"
	always_follows_primary_heir = yes
	de_jure_drift_disabled = @not_real

	male_names = { Alexander "Anastasius" Benedictus Caelestinus Callistus Clemens Eugenius Leo Gregorius Hadrianus Honorius Innocentius Ioannes Lucius Marinus Martinus Nicolaus Sergius Silvester Stephanus Urbanus Victor }
}