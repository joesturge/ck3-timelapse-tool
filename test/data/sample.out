@always_primary_score